### PR TITLE
chore(add): onboard wave-b projects to Pre-Prod

### PR DESCRIPTION
**Pull request by carslen**
Onboard wave-b products to pre-prod env. With that all wave-b products are setup as ArgoCD projects

### CODE REVIEW

File: environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   annotations:
-    avp.kubernetes.io/path: "devsecops/data/avp-config/item-relationship-service-frontend"
+    avp.kubernetes.io/path: "devsecops/data/avp-config/devsecops" # Access to 2 different secret engines may be required
```
Comment (almadigabor) [inline environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml:7]: Should we create a specific avp-config secret to access just those 2 engines?

File: environments/pre-prod/argo-projects/product-autosetup.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  roles:
+    - name: project-read-only
+      description: provide read-only access to project
+      policies:
+        - p, proj:project-autosetup:project-read-only, applications, get, project-autosetup/*, allow
+        - p, proj:project-autosetup:team-admin, exec, create, project-autosetup/*, allow
+      groups:
+        - catenax-ng:product-dft
+        - catenax-ng:release-management
+        - catenax-ng:test-management
```
Comment (almadigabor) [inline environments/pre-prod/argo-projects/product-autosetup.yaml:35]:
```suggestion
        - catenax-ng:test-management
        
```

File: environments/pre-prod/argo-projects/product-dft-sde.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  roles:
+    - name: project-read-only
+      description: provide read-only access to project
+      policies:
+        - p, proj:project-sde:project-read-only, applications, get, project-sde/*, allow
+        - p, proj:project-sde:team-admin, exec, create, project-sde/*, allow
+      groups:
+        - catenax-ng:product-dft
+        - catenax-ng:release-management
+        - catenax-ng:test-management
```
Comment (almadigabor) [inline environments/pre-prod/argo-projects/product-dft-sde.yaml:35]:
```suggestion
        - catenax-ng:test-management

```

File: environments/pre-prod/argo-projects/product-semantic-hub.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  roles:
+    - name: project-read-only
+      description: provide read-only access to project
+      policies:
+        - p, proj:project-semantic-hub:project-read-only, applications, get, project-semantic-hub/*, allow
+        - p, proj:project-semantic-hub:team-admin, exec, create, project-semantic-hub/*, allow
+      groups:
+        - catenax-ng:product-semantics
+        - catenax-ng:release-management
+        - catenax-ng:test-management
```
Comment (almadigabor) [inline environments/pre-prod/argo-projects/product-semantic-hub.yaml:35]:
```suggestion
        - catenax-ng:test-management

```

File: environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml (hunk 1 above)
Comment (almadigabor) [inline environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml:7]: Should we create a specific avp-config secret to access just those 2 engines?

**Review by almadigabor — CHANGES_REQUESTED** [review]
Some new lines are missing from the end of the files. Also added question about secret engine access handling.

File: environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml (hunk 1 above)
Comment (carslen) [reply environments/pre-prod/avp-secrets/item-relationship-service-vault-secret.yaml]: Fixed that with new commit

**Review by almadigabor — APPROVED** [review]
Looks good

**Pull request merged**